riemann_client/client.py

```diff
@@ -152,6 +153,9 @@ def create_dict(event):
                 value = list(value)
             elif descriptor.name == 'attributes':
                 value = dict(((a.key, a.value) for a in value))
+            elif descriptor.type == \
+                    google.protobuf.descriptor.FieldDescriptor.TYPE_FLOAT:
+                value = round(value, 1)
```

Comment:
The test is still good, but maybe some thought should be put into what the precision of a text float should be (then instead of comparing things non-representable accurately by a python float, it can be compared via epsilon, e.g. | a - b | < some_small_value).